packages/sanity/src/structure/comments/src/constants.ts

```diff
@@ -0,0 +1,38 @@
+import {CommentReactionOption, CommentReactionShortNames} from './types'
+
+// These should not be alphabetized as the current order is intentional
+export const COMMENT_REACTION_OPTIONS: CommentReactionOption[] = [
+  {
+    shortName: ':eyes:',
+    title: 'Eyes',
+  },
+  {
+    shortName: ':+1:',
+    title: 'Thumbs up',
+  },
+  {
+    shortName: ':rocket:',
+    title: 'Rocket',
+  },
+  {
+    shortName: ':heart_eyes:',
+    title: 'Heart eyes',
+  },
+  {
+    shortName: ':heavy_plus_sign:',
+    title: 'Heavy plus sign',
+  },
+  {
+    shortName: ':-1:',
+    title: 'Thumbs down',
+  },
+]
+
+export const COMMENT_REACTION_EMOJIS: Record<CommentReactionShortNames, string> = {
```

Comment:
any reason why this is not part of the `COMMENT_REACTION_OPTIONS` array? 
Have we considered exposing this into the plugin as configurable? Maybe it will be easier if it's all in one array.